develop/devguide/Connector/AdvancedDataMinerDataPersistenceNoSqlCassandra.md

```diff
@@ -610,15 +610,15 @@ This table is defined as follows:
 
 The DataMiner Analytics features store and maintain model data and extracted insights data in the following tables:
 
-- [analytics_alarmfocus](#analytics_alarmfocus)
+- [ai_alarmfocus](#ai_alarmfocus) (or *analytics_alarmfocus* prior to DataMiner 10.4.0/10.4.3<!-- RN 38184 -->)
 - [analytics_arrowwindows](#analytics_arrowwindows)
 - [analytics_changepoints](#analytics_changepoints)
-- [ai_cpalarms](#ai_cpalarms) (or *analytics_changepointalarmentries* prior to DataMiner 10.3.0 [CU5]/10.3.8)
-- [analytics_parameterinfo](#analytics_parameterinfo)
+- [ai_cpalarms](#ai_cpalarms) (or *analytics_changepointalarmentries* prior to DataMiner 10.4.0 [CU1]/10.4.4<!-- RN 38184 -->)
```

Comment:
@SkylineDennisDr, could you confirm that this RN will be included in 10.4.0 [CU1]? Right now, it's set to 10.5.0.